src/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './PageNotFound.scss';
+
+export const PageNotFound: React.FC = () => (
+  <>
+    <div className="wrapper">
+      <h1 className="pnf__title">404</h1>
+      <h1 className="pnf__title">PAGE NOT FOUND</h1>
+    </div>
+  </>
```

Comment:
don't use fragment if you already have a wrapper inside the component 

```suggestion
    <div className="wrapper">
      <h1 className="pnf__title">404</h1>
      <h2 className="pnf__title">PAGE NOT FOUND</h1>
    </div>
```

Reply:
also use h1 elements once per page

https://techstacker.com/seo-only-use-h1-elements-once-per-page/